ci/before_deploy.sh

```diff
@@ -0,0 +1,26 @@
+set -ex
+
+main() {
+    local src=$(pwd) \
+          stage=
+
+    case $TRAVIS_OS_NAME in
+        linux)
+            stage=$(mktemp -d)
+            ;;
+    esac
+
+    test -f Cargo.lock || cargo generate-lockfile
+
+    cargo rustc --features $FEATURES --target $TARGET --bin $CRATE_NAME --release -- -C lto
```

Comment:
I *believe* (read: correct me if I am wrong) that `$FEATURES` should be surrounded with double-quotes because `rustc` will see `--features foo bar` as `--features foo`, `bar` (`bar` being a free-standing argument / command).

If this is true then we should double-check the `test_script` and `before_deploy` blocks in `appveyor.yml`.

Reply:
Nice catch. I've did `"$FEATURES"` elsewhere in the travis script. You're right that we should use quotes so that you can just change `FEATURES` to say `clap,rand` when you need the `rand` crate.

This would break a non quoted `$FEATURES` statement.